pkgs/development/python-modules/speechbrain/default.nix

```diff
@@ -67,5 +68,7 @@ buildPythonPackage rec {
     changelog = "https://github.com/speechbrain/speechbrain/releases/tag/v${version}";
     license = lib.licenses.asl20;
     maintainers = with lib.maintainers; [ GaetanLepage ];
+    # See https://github.com/NixOS/nixpkgs/issues/466092
+    broken = stdenv.hostPlatform.isDarwin;
```

Comment:
```suggestion
    badPlatforms = [
      # See https://github.com/NixOS/nixpkgs/issues/466092
      lib.systems.inspect.patterns.isDarwin
    ];
```

Reply:
Nice! I''ll have to copy that across multiple PRs.

Reply:
Done.